### DISCUSSION TITLE
Any plans to build a iOS version of your app/service?

### DISCUSSION THREAD

**Discussion by BKrnk1337** [tvler/streetpass#36] [Q&A]
Do you plan to build an iOS version? Like a safari extension?

**Comment by tvler** [comment]
Pulling this over from a discussion I had earlier today here https://mastodon.social/@tvler/111189847613931616

> this is definitely an interesting idea. the way streetpass works right now is based on the assumption that browser extensions and notifications live in your toolbar and are easily accessible, which isn’t the case for iOS. I think the “StreetPass list” and the notification UI should probably exist as an app on your homescreen, which requires a lot of research and work to get right. It's definitely a super interesting idea that I hope to get to tho

**Reply by tvler** [reply]
This is gonna take a large amount of eng work, i'm going to think for a bit on the roadmap of this product and what features would best benefit the most users. this is still 100% on the table and very interesting to me. just need to think more!

**Reply by elight** [reply]
"Lot of eng work". Oof. Not sure what I can offer but any sense where you could use support and/or additional hands or brains?

**Reply by calebhearth** [reply]
(Hi, Evan!)

@tvler one thought is that if some sync service (iCloud sync?) were used, the apps could sync the "passes" and the main interface could continue to be the browser extension while the way that the mobile app should look is sussed out and built. That approach allows for data collection that can later be useful in the mobile app.

**Reply by tvler** [reply]
That’s a good idea! Another think I’m going to look into is if an app can just straight up read/write to Safari extension data directly

**Reply by elight** [reply]
(Small world 😃)
